adagrams/game.py

```diff
@@ -1,11 +1,130 @@
+from operator import le
```

Comment:
What is the purpose of this module? Was unintentionally it added by VS Code autocomplete? Would recommend deleting this line if the module is not used.